parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -210,7 +210,7 @@ private CompressionCodecName codec() {
             .setKeyValueMetadata(metadata)
             .setWriteSupport(getWriteSupport(type))
             .withCompressionCodec(codec())
-            .withWriteMode(ParquetFileWriter.Mode.OVERWRITE) // TODO: support modes
+            .withWriteMode(ParquetFileWriter.Mode.OVERWRITE)
```

Comment:
I think this TODO item is still valid.

This comment refers to supporting modes passed into the Parquet write builder. That's different from #302 because it isn't a table property.

Reply:
Made write mode configurable via write builder, I guess this was meant by todo.

Reply:
Great! That's exactly what the TODO meant.